src/DetailsView/details-view-container.tsx

```diff
@@ -164,9 +157,9 @@ export class DetailsViewContainer extends React.Component<DetailsViewContainerPr
         return (
             <DetailsViewOverlay
                 deps={deps}
-                previewFeatureFlagsHandler={this.props.previewFeatureFlagsHandler}
-                scopingActionMessageCreator={this.props.scopingActionMessageCreator}
-                inspectActionMessageCreator={this.props.inspectActionMessageCreator}
+                previewFeatureFlagsHandler={this.props.deps.previewFeatureFlagsHandler}
```

Comment:
Seems like we can just use `deps` here instead of `this.props.deps`

Reply:
do you mean changing the current
```typescript
const { deps } = this.props;
...
deps={deps}
```
for
```typescript
deps={this.props.deps}
```
?

Reply:
Will take care of this on a subsequent PR (there are more places we may want to look for this same pattern so...)